pkg/api/server/register_images.go

```diff
@@ -1014,8 +1014,12 @@ func (s *APIServer) registerImagesHandlers(r *mux.Router) error {
 	//     type: boolean
 	//     default: false
 	//   - in: query
-	//     name: credentials
-	//     description: "username:password for the registry"
+	//     name: password
```

Comment:
Are these additions correct at all? AFAICS the credentials are passed using a HTTP header.

Reply:
It certainly seems to authenticate using the `X-Registry-Auth` value returned from `auth.MakeXRegistryAuthHeader()`.
I think the descriptions of `password` and `username` are unnecessary, so I'll remove them.

https://github.com/containers/podman/blob/25f976e276cc102484a034e6539ba3bc91dd11ec/pkg/bindings/images/pull.go#L44